src/main/webapp/trigger/help-CommitMessageContains.html

```diff
@@ -0,0 +1,8 @@
+<p>
+	Trigger Build if commit message matches a regular expression.
+</p>
+<p>
+	If not empty, trigger if the commit message matches the regular expression.
+	The regular expression must be in the format that can be parsed by
+	<b>java.util.regex.Pattern</b>.
```

Comment:
```suggestion
	<a href="https://docs.oracle.com/javase/8/docs/api/java/util/regex/Pattern.html#sum">Java Regex.</a>	
```
Or maybe some better wording. But the link is good to have for users not well versed in the Java platform.

Reply:
Makes sense. I was just following previous wording.

Reply:
This has been updated.